### DISCUSSION TITLE
The verification slider failed on the login page of www.meituan.com

### DISCUSSION THREAD

**Discussion by JerryJian** [cefsharp/CefSharp#4016] [Q&A]
Hi,

Step to produce:
1. open the login page of meituan: https://passport.meituan.com/account/unitivelogin
2. click "手机动态码登录"
3. input a phone number of china
4. click "获取手机动态码", the verification slider will appear on most computers.
5. slider to verify, failed.

The verification successful on Microsoft Edge, Chrome, IE, only failed on CEF, anyone can help, thanks.

**Comment by amaitland** [comment]
As I said on `Gitter` you are probably better off looking for a different browser.

`CEF` only supports a subset of the `Chromium` features.

If you want to debug this then [DevTools](https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#javascript-debugging) is the place to start.

**Reply by JerryJian** [reply]
Yes, finally I use another browser and solve the problem. 
I post a discussion here to see if anyone knowns how to configure cef to solve the problem.
